src/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from "react"
+
+import { darken } from "../utils"
+import styled from "../utils/styled"
+
+export interface Value {
+  label: React.ReactNode
+  value: string
+}
+
+export interface ToggleProps {
+  /** Options available */
+  options: [Value, Value]
+  /** Current value */
+  value: Value["value"]
+  /** Callback trigger on any changes */
+  onChange: (newValue: Value["value"]) => void
+  /** Condensed option */
+  condensed?: boolean
+}
+
+const Container = styled("div")`
+  > :not(:last-child) {
```

Comment:
Why are we using `:not` rules here for first and last children given that container only _always_ contains 2 children at any given time? 🤔 

`:not(:last-child)` could be refactored as

```suggestion
  > :first-child {
```

since in this case, if it is not last, it is _definitely_ first.

Reply:
It's true, it was in prevision of the future (2+ cases) for now it's a bit over-enginering 😅 Since it works and the performance is not really impacted here (only 2 elements), I propose to keep this like this for now 👼

Reply:
It's more about the intension -> if you are not the last one: remove the radius